catalyst_voices/uikit_example/web/index.html

```diff
@@ -31,6 +31,7 @@
 
   <title>uikit_example</title>
   <link rel="manifest" href="manifest.json">
+  <script type="application/javascript" src="/assets/packages/flutter_inappwebview_web/assets/web/web_support.js" defer ></script>
```

Comment:
My IDE formats it a bit differently, like it:

```
  <script type="application/javascript" src="/assets/packages/flutter_inappwebview_web/assets/web/web_support.js"
    defer></script>
```

Would prefer that we have the same formatting rules otherwise we will just overwrite each other's changes 🙃